tutorials/preprocessing/90_eyetracking_data.py

```diff
@@ -101,123 +108,114 @@
 first_cal.plot(show_offsets=True)
 
 # %%
-# Get stimulus events from DIN channel
-# ------------------------------------
+# Extract common stimulus events from the data
+# --------------------------------------------
```

Comment:
I wonder if this whole section is out of place; did you consider waiting to explain this until just before / at the beginning of the "aligning..." section?  Is there value in explaining here/showing the "flash" events in the timeseries plot? (to me it seems like it's not crucial since you're mostly talking about interpolation there)

Reply:
Totally, I actually tried moving it and then realized the plot method was using the events.

I do like having the flash event in the time series plot - but we do plot the time series again after alignment, so I'll just move this section and forego including the flash event in the first time series.